docs/experiments.md

```diff
@@ -0,0 +1,30 @@
+# Experiments
+
+Fenix uses Mozilla's [Experiments service for experimentation](https://github.com/mozilla-mobile/android-components/blob/master/components/service/experiments/README.md).
+
+## Experiment Naming
+
+Naming must be unique and under 100 characters.
+
+Name experiments in the format of "fenix_experiment_issue", such as "fenix_etp_5651"
```

Comment:
If experiment titles include multiple words, should they be snake cased or camel cased?  Would you mind specifying, and maybe updating the example to show it as well?  Maybe `fenix_buttonColor_1`, or `fenix_button_color_1`.